cachalot/tests/read.py

```diff
@@ -896,9 +896,7 @@ def test_extra_order_by(self):
         self.assert_query_cached(qs, [self.t2, self.t1])
 
     def test_table_inheritance(self):
-        with self.assertNumQueries(
-            3 if self.is_sqlite else (4 if DJANGO_VERSION >= (4, 2) else 2)
-        ):
+        with self.assertNumQueries(2):
```

Comment:
Could you help me understand why it's only 2 now? Were you only testing on Django 4.2 and only on non SQLite databases? Or is this a thing with Django internals?

Reply:
Actually, I just removed all those ifs. SQLite does still work with only 2 now, because it also adds a 'BEGIN' to the transaction list. I was testing multiple combinations of Python, Django, and db, but couldn't run all of them at one.